angular/projects/spark-angular/src/lib/components/inputs/sprk-selection-item-container/sprk-selection-item-container.component.spec.ts

```diff
@@ -76,4 +80,23 @@ describe('SprkSelectionItemContainerComponent', () => {
       selectionInputElement.id
     );
   });
+
+  it('should add the checkbox classes to the input and the label if the type is checkbox', () => {
+    expect(selectionInputElement.classList.contains('sprk-b-Checkbox__input'))
+      .toBe(true);
+    expect(selectionLabelElement.classList.contains('sprk-b-Checkbox__label'))
+      .toBe(true);
+  });
+
+  it('should add the radio classes to the input and the label if the type is radio', () => {
```

Comment:
Should add a check to make sure the id / for logic is happening.

Reply:
There was an existing test to make sure that they match, since it was existing functionality.